include/wlr/types/wlr_text_input.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2018 Purism SPC
+ *
+ * SPDX-License-Identifier: MIT
+ */
```

Comment:
hm

Reply:
Please correct me if the license is wrong - I tried to match the identifier to the LICENSE file.